CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+
+## 0.0.1 - 2019-03-11
```

Comment:
Since we have not made our first release, all of the changes will need to go under the unreleased section.

Reply:
Done